phoenix-core/src/it/java/org/apache/phoenix/end2end/ParameterizedIndexUpgradeToolIT.java

```diff
@@ -346,6 +362,28 @@ public void testDryRunAndFailures() throws Exception {
         }
     }
 
+    @Test
+    public void verifyViewAndViewIndexes() throws SQLException {
```

Comment:
Does this test belong in a different class? Doesn't actually seem related to index upgrades, but to looking up metadata from syscat.

Reply:
Sure. :)

Reply:
I kept it in this file, as it is testing the code in IUT.

Reply:
@gjacoby126 Where do you propose to keep this test? I was considering "NonParameterizedIndexUpgradeToolIT". any thoughts?